native/packages/react-native-bpk-component-switch/readme.md

```diff
@@ -0,0 +1,48 @@
+# react-native-bpk-component-switch
+
+> Backpack React Native switch component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-switch --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import BpkSwitch from 'react-native-bpk-component-switch';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
```

Comment:
Can we destruct only the tokens we need instead?

Reply:
Even in the readme? We're not doing that in any other readmes.

Reply:
Seems like a good thing to encourage imo

Reply:
Fair enough, but if we're gonna do that we should probably change all the readmes at once in another PR.

Reply:
Agreed 👍  can we create a task for this please? 😇

Reply:
https://gojira.skyscanner.net/browse/BPK-1111 🙌